applications/StructuralMechanicsApplication/custom_python/add_custom_strategies_to_python.cpp

```diff
@@ -109,11 +111,10 @@ void  AddCustomStrategiesToPython(pybind11::module& m)
     class_< EigensolverStrategyType, typename EigensolverStrategyType::Pointer,BaseSolvingStrategyType >(m,"EigensolverStrategy")
     .def(init<ModelPart&, BaseSchemeType::Pointer, BuilderAndSolverPointer>() )
             ;
-             
 
     class_< FormfindingUpdatedReferenceStrategyType,typename FormfindingUpdatedReferenceStrategyType::Pointer, BaseSolvingStrategyType >(m,"FormfindingUpdatedReferenceStrategy")
```

Comment:
I think BaseSolvingStrategyType should be ResidualBasedNewtonRaphsonStrategyType

Reply:
done